types/shelljs/index.d.ts

```diff
@@ -537,10 +544,28 @@ export interface ShellConfig {
     fatal: boolean;
 
     /**
+     * Will print each executed command to the screen. Default is true.
+     * @type {boolean}
+     */
+    verbose: boolean;
+
+    /**
+     * Passed to glob.sync() instead of the default options ({}).
+     * @type {Object}
+     */
+    globOptions: GlobOptions;
+
+    /**
      * Absolute path of the Node binary. Default is null (inferred).
      * @type {string|null}
      */
     execPath: string | null;
+
+    /**
+     * Reset shell.config to the defaults.
+     * @type {ResetConfigFunction}
+     */
+    reset: ResetConfigFunction;
```

Comment:
Oh, in that case I am not sure what the type in the comment should be, though. I guess that's why I went for a new type.